setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = pychart-lang
-version = 0.0.0
+version = attr: pychart.__version__
```

Comment:
Nice! Maybe its not relevant here, but probably will be on the backend: does python have a way to store/load environment secrets without checking into git?

Javascript has a file `.env` that behaves like this so I'm wondering if python has the same or if you need to do this and then gitignore config/__init__.py

Reply:
Yes! You can access/modify environment variables using:
`import os

os.environ['KEY'] = 'value'`